yjit_backend.c

```diff
@@ -367,6 +461,17 @@ void test_backend()
         ir_ret(jit);
     })
 
+    TEST("adding with an immediate and a register", 7, {
+        ir_opnd_t opnd1 = ir_mov(jit, IR_REG(RAX), ir_imm(3));
+        ir_mov(jit, IR_REG(RAX), ir_add(jit, ir_imm(4), opnd1));
+        ir_ret(jit);
+    })
+
+    TEST("adding with both immediates", 7, {
+        ir_mov(jit, IR_REG(RAX), ir_add(jit, ir_imm(3), ir_imm(4)));
+        ir_ret(jit);
+    })
+
```

Comment:
Could you add a third test with two operations here? Are you able to handle that at this point?

Eg: RAX = 3 + 7 - 1